tests/database/test_model.py

```diff
@@ -116,8 +119,8 @@ class TestSessionNode(ModelTestBase):
     attrs = {"distro_type": "CentOS", "distro_version": "8Stream"}
 
     def _db_obj_get_dependencies(self):
-        project = model.Project(name="World Domination", ssh_key="Muahahahaha!")
-        session = model.Session(project=project)
+        tenant = model.Tenant(name="World Domination", is_admin=True, ssh_key="Muahahahaha!")
```

Comment:
👍😁

Reply:
I reckon they need that admin access, if they want to dominate the world and so is the column. ;P